nntrainer/tensor/swap_device.cpp

```diff
@@ -88,7 +89,7 @@ void *SwapDevice::getBuffer(off_t offset, size_t size, bool alloc_only) {
       << "SwapDevice: seek file: " << dev_path;
 
     len = read(fd, ptr, size);
-    NNTR_THROW_IF(len != (ssize_t)size, std::runtime_error)
+    NNTR_THROW_IF(len != (size_t)size, std::runtime_error)
```

Comment:
do we need to cast 'ssize_t' to 'size_t'?
because it looks 'read func' returns 'ssize_t' type.